registration-processor/registration-processor-registration-status-service-impl/src/test/java/io/mosip/registration/processor/status/service/RegistrationStatusServiceTest.java

```diff
@@ -220,6 +220,18 @@ public void testGetUnProcessedPackets() {
 				statusList);
 		assertEquals("REPROCESS", dtolist.get(0).getLatestTransactionStatusCode());
 	}
+	
+	@Test
+	public void testGetPausedPackets() {

```

Comment:
Change test cases according to the statusCode field, and add negative test cases for exceptions